lighthouse-core/config/default-config.js

```diff
@@ -84,7 +84,7 @@ module.exports = {
     'errors-in-console',
     'time-to-first-byte',
     'first-cpu-idle',
-    'consistently-interactive',
+    'interactive',
```

Comment:
i do prefer 'time-to-interactive' personally, but i could could live with this slug if i had to.